cmd/config.go

```diff
@@ -164,6 +166,96 @@ keyfiles written in:
 	},
 }
 
+var encryptCredentialsDataCmd = &cobra.Command{
+	Use:     "encrypt-data <encoding keyfile> <text>",
+	Aliases: []string{"encrypt-data"},
```

Comment:
I'm not sure what you want to do here. `Aliases` is for giving other names to a command, but you use the same name, so it probably does nothing.